.github/workflows/test-kind.yaml

```diff
@@ -20,11 +20,12 @@ jobs:
           k8sVersion: ${{ matrix.k8sVersion }}
     - name: Kind Cluster
       uses: helm/kind-action@0025e74a8c7512023d06dc019c617aa3cf561fde # v1.10.0
-    - name: check kind cluster
+    - name: check kind cluster and taint nodes
       shell: bash
       run: |
         kubectl config current-context
         kubectl get nodes
+        kubectl taint nodes chart-testing-control-plane CriticalAddonsOnly:NoSchedule
```

Comment:
Is there a shorter name that we can use for the chart here?

Reply:
This is the default node name used when you create a cluster with the `helm/kind-action`. We can change the name if you want.